Сетка страниц и крупных блоков на флексах

neo217 opened the pull request:
connected fonts and added main backgrounds, fixed minor bugs

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/122095/tasks/6)

:boom: https://htmlacademy-htmlcss.github.io/122095-device-33/4/

---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/33/user/122095/tasks/7)

neo217 committed:
did the basic styling of the page

connected fonts and added main backgrounds, fixed minor bugs

Viktor-Rybakov commented on styles/styles.css:
```diff
@@ -0,0 +1,482 @@
+}
+
+
+.header-search-field::-webkit-input-placeholder {
```
Такой селектор стилизует только плейсхолдер, но нужно и сам инпут стилизовать - задай для инпута параметры, которые ты перечислил, а для плейсхолдера - только цвет (хром) и опасити (фаерфокс)

Viktor-Rybakov replied:
Посмотри, какие стили введнееного текста и плейсхолдера в разных состояниях
![image](https://user-images.githubusercontent.com/53224541/152604249-971a548b-6d36-413e-a62a-a54eef2258d0.png)

Viktor-Rybakov commented on styles/styles.css:
```diff
@@ -0,0 +1,482 @@
+  line-height: 18px;
+}
+
+.main-top-product-number {
```
На макете цифры белого цвета

Viktor-Rybakov commented on styles/styles.css:
```diff
@@ -0,0 +1,482 @@
+  color: #000000;
+}
+
+.contacts-address-title, working-hours-title {
```
Забыл поставить точку у второго селектора

Viktor-Rybakov commented on styles/styles.css:
```diff
@@ -0,0 +1,482 @@
+  color: #000000;
+}
+
+.pagination-link {
```
Нужно убрать подчеркивание

Viktor-Rybakov reviewed (changes requested):
ОТличная работа! Нужно поправить немного мелочей + добавить стили текста для интерактивных элементов в разных состояниях (ховер, нажатие, фокус).

neo217 committed:
creating a grid on flex on the main page

made stylization of large blocks on flex on the main page
started small blocks on flex and inserted icons on the main page

neo217 committed:
finished layout of the main page and started layout of the inner page

completed large blocks on flex, started marking on grids, updated fonts
